index.html

```diff
@@ -72,15 +72,15 @@
 
             <div class="button_container">
               <p class="button_group_text site_font">Learn more about skydiving. </p>
-                <img class="skydiver_image" src="./assets/skydiver3.png">
+                <img class="skydiver_image grow" src="./assets/skydiver3.png">
                 <div class="button button_effect">
                   <div class="site_font button_font">The Manual</div>
                 </div>
             </div>
           </div>
         
           <div id="dropzoneButton" class="button">
-            <div class="site_font button_font find">Find a Dropzone</div>
+            <div class="site_font button_font find" onclick="window.location='../pages/map.html'">Find a Dropzone</div>
```

Comment:
So this works but it wrong bc itss not as accessible. Can you update all the div like this per this CSS tricks article? (Scroll to update 2011) https://css-tricks.com/creating-clickable-divs/